superset-frontend/src/pages/SavedQueryList/index.tsx

```diff
@@ -443,7 +447,7 @@ function SavedQueryList({
   const filters: Filters = useMemo(
     () => [
       {
-        Header: t('Name'),
+        Header: t('search'),
```

Comment:
What's the impetus for this change? All the other list views say "Name" since they only search the Name. Does this one also search descriptions? Obviously it ought to be truthful, but otherwise I'd err on the side of consistency.

Reply:
Yes, it searches all text columns (name, description, sql, label), so it can probably just be called "search"

https://github.com/apache/superset/pull/28349#discussion_r1590479993

Reply:
If you have the wherewithal, it might be nice to add an info tooltip next to the header to state exactly that, so people know exactly what it's searching. Then (on another PR) it'd be good to carry this through to other list views, if any of them indeed search more than name. 

CC @yousoph @kasiazjc in case this merits tracking.

Reply:
ack

Reply:
@rusackas I've added the tooltip & updated the screenshot in description - 

<img width="1724" alt="Screenshot 2024-05-06 at 4 25 32 PM" src="https://github.com/apache/superset/assets/5911558/3ed1cc7e-34ee-4b96-a81b-620ad2762380">